File: app/src/main/java/com/instafinancials/vendoralpha/apicall/network/homeapi/instabasic/pojo/DirectorSignatoryMasterBasic.java
package com.instafinancials.vendoralpha.apicall.network.homeapi.instabasic.pojo;

import java.io.Serializable;

public class DirectorSignatoryMasterBasic implements Serializable {
    private DirectorCurrentMasterBasic DirectorCurrentMasterBasic;

    private DirectorAllDirectorshipMasterBasic DirectorAllDirectorshipMasterBasic;

    private String LastUpdatedDateTime;

    @Override
    public String toString() {
        return "DirectorSignatoryMasterBasic{" +
                "DirectorCurrentMasterBasic=" + DirectorCurrentMasterBasic +
                ", DirectorAllDirectorshipMasterBasic=" + DirectorAllDirectorshipMasterBasic +
                ", LastUpdatedDateTime='" + LastUpdatedDateTime + '\'' +
                '}';
    }
}

File: app/src/main/java/com/instafinancials/vendoralpha/deletellater/network/pojonew/DirectorSignatoryMasterBasic.kt
package com.instafinancials.vendoralpha.deletellater.network.pojonew

import java.io.Serializable

data class DirectorSignatoryMasterBasic (

	val directorCurrentMasterBasic : DirectorCurrentMasterBasic,
	val signatoryCurrentMasterBasic : SignatoryCurrentMasterBasic
): Serializable